ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ntp.md

```diff
@@ -60,13 +60,22 @@ interface Management1
 | 10.1.1.1 | true |
 | 10.1.1.2 | - |
 
+
+|--ID--|                Hashed Key              | Algo | Trusted |
+|------|----------------------------------------|------|---------|
+|  10  |    356708192346f678a878d6e99889767a    |  md5 |   True  |
+
 ### NTP Device Configuration
 
 ```eos
 !
 ntp local-interface vrf test lo1
 ntp server vrf test 10.1.1.1 prefer
 ntp server vrf test 10.1.1.2
+
```

Comment:
This is more cosmetic, but try to have the render CLI output match as close as possible the order on EOS device. Also, remove the new line introduced.